spdata/types.py

```diff
@@ -0,0 +1,39 @@
+"""Reusable types across package"""
+
+
+import numpy as np
+
+
+class Dataset:
```

Comment:
Same here - why are we omitting Z and labels

Reply:
Not omitting anymore.